src/native/libs/System.Globalization.Native/CMakeLists.txt

```diff
@@ -135,8 +131,9 @@ if (NOT CLR_CMAKE_TARGET_BROWSER AND NOT CLR_CMAKE_TARGET_WASI AND NOT CLR_CMAKE
     set(NATIVEGLOBALIZATION_SOURCES ${NATIVEGLOBALIZATION_SOURCES} pal_timeZoneInfo.c)
 endif()
 
-if (MSVC)
-    set_source_files_properties(${NATIVEGLOBALIZATION_SOURCES} PROPERTIES LANGUAGE CXX)
+if (MSVC AND MSVC_VERSION LESS 1939)
```

Comment:
Our minimal VS version [offsets](https://learn.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance?view=msvc-160#c-standard-library-features-1) C99 support required here, so this check is unnecessary.

https://github.com/dotnet/runtime/blob/79a1c917798a1fa27455a8683657bc2789eee939/eng/native/init-vs-env.cmd#L57

Reply:
`__typeof__` support is new in MSVC 19.39: https://learn.microsoft.com/en-us/cpp/c-language/typeof-c?view=msvc-170